utils/durability-test/startup.exp

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/expect -f
+
+# Wait enough (forever) until a long-time boot
+set timeout -1
+
+spawn qemu-system-x86_64 \
+    -hda bionic-server-cloudimg-amd64.img \
```

Comment:
`-hda file=bionic-server-cloudimg-amd64.img,cache=writeback` is much more realistic.